relay-server/src/envelope.rs

```diff
@@ -142,35 +142,39 @@ impl ItemType {
             }
         }
     }
+
+    pub fn as_str(&self) -> &str {
+        match self {
+            Self::Event => "event",
+            Self::Transaction => "transaction",
+            Self::Security => "security",
+            Self::Attachment => "attachment",
+            Self::FormData => "form_data",
+            Self::RawSecurity => "raw_security",
+            Self::Nel => "nel",
+            Self::UnrealReport => "unreal_report",
+            Self::UserReport => "user_report",
+            Self::UserReportV2 => "feedback",
+            Self::Session => "session",
+            Self::Sessions => "sessions",
+            Self::Statsd => "statsd",
+            Self::MetricBuckets => "metric_buckets",
+            Self::MetricMeta => "metric_meta",
+            Self::ClientReport => "client_report",
+            Self::Profile => "profile",
+            Self::ReplayEvent => "replay_event",
+            Self::ReplayRecording => "replay_recording",
+            Self::CheckIn => "check_in",
+            Self::Span => "span",
+            Self::OtelSpan => "otel_span",
+            Self::Unknown(ref s) => s,
```

Comment:
Potential cardinality explosion here, but should be fine?

Reply:
I'd prefer to map this to unknown for the metric.